smartmin/widgets.py

```diff
@@ -44,6 +44,12 @@ def __init__(self, thumb_width=75, thumb_height=75):
         super(ImageThumbnailWidget, self).__init__({})
 
     def render(self, name, value, attrs=None, renderer=None):
+        try:
```

Comment:
I used this `try/except` so we use the thumbnails when the dependency is there and configured but we keep that as optional, if it not available we still rely on the normal URL of the object

Reply:
@norkans7 had to make small fix to this because it was blowing up when `value` isn't set https://github.com/nyaruka/smartmin/commit/42b6951cbcd895dc11bbaa6808307c420d81ed4c